src/Sql/NotificationCenter/dbo/Stored Procedures/Notification_ReadById.sql

```diff
@@ -0,0 +1,10 @@
+CREATE PROCEDURE [dbo].[Notification_ReadById]
+    @Id UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT *
+    FROM [dbo].[Notification]
```

Comment:
🎨 We should use a view everywhere when we return results. That can be a follow-up PR.

Reply:
Fixed in 34ea5a30